src/olympia/amo/management/commands/check_locales_completion_rate.py

```diff
@@ -47,12 +47,11 @@ class Command(BaseCommand):
     # that by querying amo and amoFrontend.
     PONTOON_PROJECTS = 2
```

Comment:
Question. do we currently have a CI check that locales above the threshold are included? That would be the thing I'd expect to make a PR like this pass. Does that currently exist?

Reply:
We don't have a check that calls pontoon in CI, so we don't check that all locales are below threshold in CI. This was explicitly not a goal of this project, multiple reasons for this:
- The command (and therefore the threshold) is only there to generate an email once a month. We then manually review the email to decide which locales to keep/add. We don't necessarily follow the threshold, it's more of a recommendation (though it will warn about the same locale the next month if things don't change)
- Things change quickly. New strings are added, translations are changed etc several times per week. We don't want CI to break just because we're suddenly at 79% for a given locale for a couple days, or even a couple weeks.